Counties/Florida/Bay County/Scraper/Scraper.py

```diff
@@ -0,0 +1,437 @@
+    summary_table_col1 = driver.find_elements_by_xpath('//*[@id="summaryAccordionCollapse"]/table/tbody/tr/td[1]/dl/dd')
+    summary_table_col2 = driver.find_elements_by_xpath('//*[@id="summaryAccordionCollapse"]/table/tbody/tr/td[2]/dl/dd')
+    summary_table_col3 = driver.find_elements_by_xpath('//*[@id="summaryAccordionCollapse"]/table/tbody/tr/td[3]/dl/dd')
+
+    # Wait for court dockets to load
+    try:
+        WebDriverWait(driver, 5).until(EC.presence_of_element_located((By.ID, 'gridDocketsView')))
+    except TimeoutException:
+        raise ValueError('Dockets did not load.')
+
+    charges_table = driver.find_elements_by_xpath('//*[@id="gridCharges"]/tbody/tr')
+    docket_public_defender = driver.find_elements_by_xpath(
+        "//*[contains(text(), 'COURT APPOINTED ATTORNEY') and contains(text(), 'ASSIGNED')]")
+    docket_attorney = driver.find_elements_by_xpath("//*[contains(text(), 'DEFENSE') and contains(text(), 'ASSIGNED')]")
+    docket_pleas = driver.find_elements_by_xpath("//*[contains(text(), 'PLEA OF')]")
+    docket_attachments = driver.find_elements_by_class_name('casedocketimage')
+
+    _id = str(uuid.uuid4())
+    _state = settings['state-code']
+    _county = settings['county']
+    CaseNum = summary_table_col2[1].text.strip()
+    AgencyReportNum = summary_table_col1[4].text.strip()
+    ArrestDate = None  # Can't be found on this portal
+    FilingDate = summary_table_col1[2].text.strip()
+    OffenseDate = None  # Can't be found on this portal
+    DivisionName = summary_table_col3[3].text.strip()
+    CaseStatus = summary_table_col3[1].text.strip()
+
+    if settings['collect-pii']:
+        # Create list of assigned defense attorney(s)
+        defense_attorney_text = list(map(lambda x: x.text, docket_attorney))
+        DefenseAttorney = ScraperUtils.parse_attorneys(defense_attorney_text)
+        # Create list of assigned public defenders / appointed attorneys
+        public_defender_text = list(map(lambda x: x.text, docket_public_defender))
+        PublicDefender = ScraperUtils.parse_attorneys(public_defender_text)
+        # Get Judge
+        Judge = summary_table_col1[0].text.strip()
+
+        # Download docket attachments.
+        if settings['save-attachments']:
+            for attachment_link in docket_attachments:
+                attachment_docket_text = attachment_link.find_element_by_xpath('./../../td[3]').text.strip()
+                # Download will start in a new
+                main_window = driver.current_window_handle
+                ScraperUtils.save_download(output_attachments, attachment_link.click,
+                                           '{}-{}'.format(case_number, attachment_docket_text))
+                driver.switch_to.window(driver.window_handles[-1])
+                driver.close()
+                driver.switch_to.window(main_window)
+
+    else:
+        DefenseAttorney = []
+        PublicDefender = []
+        Judge = None
+
+    Charges = {}
```

Comment:
Why upper-case var name?

Reply:
It's stylistic so that they match the CSV to represent which columns they would be added to. I know it's not python convention. It can be changed to lowercase.